custom/conf/app.example.ini

```diff
@@ -617,6 +617,8 @@ REGISTER_MANUAL_CONFIRM = false
 ; List of domain names that are allowed to be used to register on a Gitea instance
 ; gitea.io,example.com
 EMAIL_DOMAIN_WHITELIST =
+; Comma-separated list of domain names that are not allowed to be used to register on a Gitea instance
+EMAIL_DOMAIN_BLACKLIST =
```

Comment:
there where some complains about blacklist vs blocklist ... lets just stick with **blocklist** ...

Reply:
Fine with me. I have used *blacklist* solely for consistency with the `BLACKLIST_URIS` setting for OpenID.

Reply:
yes keep the old one since it would break things but if we can avoid blacklist ...